mleap-xgboost-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/mleap/XGBoostClassificationModelParitySpec.scala

```diff
@@ -106,7 +129,6 @@ class XGBoostClassificationModelParitySpec extends FunSpec
             }
             assert(Math.abs(v2 - v1) < 0.0000001)
```

Comment:
Magic number in code :S
Could this maybe be 
`SEVEN_SIGNIFICANT_DIGITS = 0.0000001`